nosql/1.0/_index.md

```diff
@@ -16,45 +16,28 @@ https://github.com/eclipse-ee4j/nosql/pull/55
+* [Jakarta NoSQL 1.0 Specification Document TCK](https://github.com/eclipse/jnosql/archive/1.0.0-b3.zip)([sig](),[sha](),[pub]())
 * Maven coordinates
-  * [jakarta.nosql.communication:communication-core:jar:1.0.0-b2](https://repo1.maven.org/maven2/jakarta/nosql/communication/communication-core/1.0.0-b2/)
-  * [jakarta.nosql.communication:communication-column:jar:1.0.0-b2](https://repo1.maven.org/maven2/jakarta/nosql/communication/communication-column/1.0.0-b2/)
-  * [jakarta.nosql.communication:communication-document:jar:1.0.0-b2](https://repo1.maven.org/maven2/jakarta/nosql/communication/communication-document/1.0.0-b2/)
-  * [jakarta.nosql.communication:communication-key-value:jar:1.0.0-b2](https://repo1.maven.org/maven2/jakarta/nosql/communication/communication-key-value/1.0.0-b2/)
-  * [jakarta.nosql.mapping:mapping-core:jar:1.0.0-b2](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-core/1.0.0-b2/)
-  * [jakarta.nosql.mapping:mapping-key-value:jar:1.0.0-b2](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-key-value/1.0.0-b2/)
-  * [jakarta.nosql.mapping:mapping-column:jar:1.0.0-b2](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-column/1.0.0-b2/)
-  * [jakarta.nosql.mapping:mapping-document:jar:1.0.0-b2](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-document/1.0.0-b2/)
-  * [jakarta.nosql.mapping:mapping-api:jar:1.0.0-b2](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-api/1.0.0-b2/)
+  * [jakarta.nosql.communication:communication-core:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/communication/communication-core/1.0.0-b3/)
+  * [jakarta.nosql.communication:communication-column:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/communication/communication-column/1.0.0-b3/)
+  * [jakarta.nosql.communication:communication-document:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/communication/communication-document/1.0.0-b3/)
+  * [jakarta.nosql.communication:communication-key-value:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/communication/communication-key-value/1.0.0-b3/)
+  * [jakarta.nosql.mapping:mapping-core:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-core/1.0.0-b3/)
+  * [jakarta.nosql.mapping:mapping-key-value:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-key-value/1.0.0-b3/)
+  * [jakarta.nosql.mapping:mapping-column:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-column/1.0.0-b3/)
+  * [jakarta.nosql.mapping:mapping-document:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-document/1.0.0-b3/)
+  * [jakarta.nosql.mapping:mapping-api:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-api/1.0.0-b3/)
 
 # Compatible Implementations
 
 * [Eclipse JNoSQL](http://www.jnosql.org/)
-
```

Comment:
The review records should not be removed. Please add this back in.

Reply:
@ivargrimstad done, thank you